docs/docsite/rst/dev_guidelines.rst

```diff
@@ -480,20 +480,135 @@ When you make a call using boto3, you will probably get back some useful informa
 should return in the module.  As well as information related to the call itself, you will also have
 some response metadata.  It is OK to return this to the user as well as they may find it useful.
 
-Boto3 returns all values CamelCased.  Ansible follows Python standards for variable names and uses
-snake_case. There is a helper function in module_utils/ec2.py called ``camel_dict_to_snake_dict``
-that allows you to easily convert the boto3 response to snake_case.
+Boto3 returns most keys in CamelCase.  Ansible adopts python standards for naming variables and usage.
+There is a useful helper function called ``camel_dict_to_snake_dict`` that allows for an easy conversion
+of the boto3 response to snake_case.  It resides in ``module_utils/common/dict_transformations``.
```

Comment:
```suggestion
of the boto3 response to snake_case.  It resides in ``ansible.module_utils.``.
```

The file isn't in this collection, so providing a file path seems a little strange.

Reply:
haha oops! thanks for catching